File: compiled_lang/cpp/src/e20.branch.cpp
#include <iostream>
#include <string>

int main() {
    std::cout << "Input a number: ";
    std::string line;
    std::getline(std::cin, line);
    int number = std::stoi(line);

    if (number > 0)
        std::cout << "Number is greater than 0" << std::endl;
    else if (number < 0)
        std::cout << "Number is less than 0" << std::endl;
    else
        std::cout << "Number is 0" << std::endl;

    return 0;
}

File: compiled_lang/cs/src/e20.branch.cs
using System;

class E20Branch
{
    static void Main()
    {
        Console.Write("Input a number: ");
        int number = int.Parse(Console.ReadLine());

        if (number > 0)
            Console.WriteLine("Number is greater than 0");
        else if (number < 0)
            Console.WriteLine("Number is less than 0");
        else
            Console.WriteLine("Number is 0");
    }
}
